py/semaphore/consts.py

```diff
@@ -0,0 +1,21 @@
+from .processing import StoreNormalizer
+
+__all__ = ["SPAN_STATUS_MAPPING"]
+
+
+def _get_span_status_mapping():
+    normalizer = StoreNormalizer()
+    rv = {}
+
+    for i in range(255):
+        name = normalizer.normalize_event({"contexts": {"trace": {"status": i,},},})[
```

Comment:
This is quite extreme :) Just wondering, exporting the enum directly via the CABI, or exposing a function that returns a static name was not an option?